scripts/build_components.sh

```diff
@@ -97,6 +97,8 @@ for dir in "${components_to_build[@]}"; do
   docker build --push "${args[@]}" \
    --build-arg="FONDANT_VERSION=${tags[0]}" \
    --label org.opencontainers.image.source=https://github.com/${namespace}/${repo} \
+   --progress=plain \
+   --no-cache \
```

Comment:
This should definitely not be there. We want to keep caching when building components

Reply:
This still needs to be addressed